utils/flags/flags.go

```diff
@@ -45,21 +45,32 @@ func GetCliFlags() []cli.Flag {
 		&cli.StringFlag{
 			Name:    "host",
 			Value:   "",
-			Usage:   "Address to listen on. Listens on all network interfaces by default.",
+			Usage:   "DEPRECATED. Use --http_address to specify the HTTP server listener.",
 			EnvVars: []string{"BAZEL_REMOTE_HOST"},
 		},
 		&cli.IntFlag{
 			Name:    "port",
 			Value:   8080,
-			Usage:   "The port the HTTP server listens on.",
+			Usage:   "DEPRECATED. Use --http_address to specify the HTTP server listener.",
 			EnvVars: []string{"BAZEL_REMOTE_PORT"},
 		},
+		&cli.StringFlag{
+			Name:    "http_address",
+			Usage:   "Address specification for the HTTP server listener, formatted either as [host]:port for TCP or unix://path.sock for Unix domain sockets.",
+			EnvVars: []string{"BAZEL_REMOTE_HTTP_ADDRESS"},
+		},
 		&cli.IntFlag{
 			Name:    "grpc_port",
 			Value:   9092,
-			Usage:   "The port the gRPC server listens on. Set to 0 to disable.",
```

Comment:
I think we should still mention that 0 disables grpc.

Reply:
Fixed in 877903af3b08f8f350f1f69336a65e9e11afcb50.